app/views/mailer/mfa_recommendation_announcement.html.erb

```diff
@@ -0,0 +1,126 @@
+<% @title = "Enable multi-factor authentication (MFA) on your RubyGems account" %>
```

Comment:
lol! This shouldn't be here. I must've accidentally committed this. I'll clean up that commit later today.

Reply:
Deleted!